src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/KravgrunnlagMottaker.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.tilbake.kravgrunnlag
+
+import no.nav.familie.prosessering.domene.Task
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Profile
+
+import org.springframework.jms.annotation.JmsListener
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import javax.jms.TextMessage
+
+import no.nav.familie.prosessering.domene.TaskRepository
+import java.util.*
```

Comment:
Jeg oppdaterte code formatter i familie-settings for å unngå wildcard imports. Kan du gjerne bruke den?

Reply:
👍  Done